doc/articles/uno-notimplemented-types.md

```diff
@@ -0,0 +1,14 @@
+## The member/type is not implemented
+
+Uno Platform is built around providing the full APIs provided by WinRT and WinUI, by replicating the available surface to be used in Uno Platform apps.
+
+This surface is very large and with the help of the community, the Uno Platform team is hard at work to add support for APIs that are marked as "Not Implemented".
```

Comment:
This WinUI & WinRT surface is very large. We covered a great portion of it since 2013, to the level of even publicly-listed enterprises deploying apps with it. To prioritize the remainder of unimplemented WinUI & WinRT API surface, we need community help to add support for APIs that are marked as "Not Implemented".